src/Bicep.LangServer/BicepCompilationManager.cs

```diff
@@ -44,45 +44,45 @@ public BicepCompilationManager(ILanguageServerFacade server, ICompilationProvide
 
         public void RefreshCompilation(DocumentUri documentUri)
```

Comment:
>RefreshCompilation [](http://example.com/codeflow?start=20&length=18)

I haven't implemented the logic to figure out which documents need to be recompiled based ont he module refs they have. That will come in a separate PR.